app/models/olympian.rb

```diff
@@ -11,4 +11,19 @@ class Olympian < ApplicationRecord
   validates :height, presence: true
   validates :weight, presence: true
   validates :team, presence: true
+
+  # instance methods
```

Comment:
`.youngest` and `.oldest` are very similar, it might be worth it to combine them into one method that takes an argument.